### PR TITLE
post request

### PR DESCRIPTION
**Pull request by Haris-Siddiqui**

### CODE REVIEW

File: src/api/post.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+import config from "./config";
+
+const url = config.BASE_URL;
+
+export const handleAddPost = (text, mediaUrl) => {
+  const id = "64d679342e7340553804ccdf";
+  console.log(id);
+//   return config.makeRequest(() => {
```
Comment (i-abdul-raheem) [inline src/api/post.js]: Remove all comments

File: src/components/CreatePostWrapper/CreatePost.jsx [hunk 2]
```diff
@@ -1,4 +1,10 @@
-export const CreatePost = () => {
+import { useState } from "react";
+
+export const CreatePost = ({setText}) => {
+   
+   const handleTextChange = (e) => {
+    setText(e.target.value);  };
+  
   return (
     <div className='page-content'>
```
Comment (i-abdul-raheem) [inline src/components/CreatePostWrapper/CreatePost.jsx:8]: Please prettify all the code

File: src/components/CreatePostWrapper/CreatePostWrapper.jsx [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
+import { Footer } from "./Footer";
+import { Header } from "./Header";
+import "./style.css";
+import { handleAddPost, handleGetPosts } from "../../api/post";
 
 export const CreatePostWrapper = () => {
+  const [text, setText] = useState("");
+  const [mediaUrl, setMediaUrl] = useState("");
+  const handlePost = () => {
+    // handleGetPosts();
```
Comment (i-abdul-raheem) [inline src/components/CreatePostWrapper/CreatePostWrapper.jsx]: remove comments

Comment (i-abdul-raheem) [reply src/components/CreatePostWrapper/CreatePostWrapper.jsx]: remove logs

File: src/components/CreatePostWrapper/Footer.jsx [hunk 4]
```diff
@@ -1,44 +1,20 @@
...
+    if (selectedFile) {
+      setName(selectedFile.type.startsWith('image/') ? 'image' : 'video');
+      setMediaUrl(URL.createObjectURL(selectedFile));
     }
   };
 
   return (
     <footer className="footer border-0 fixed">
-      <div className="container" style={name==="video"?{position:"relative"}:{}}>
+      <div className="container" style={name === 'video' ? { position: 'relative' } : {}}>
```
Comment (i-abdul-raheem) [inline src/components/CreatePostWrapper/Footer.jsx]: use null instead
`{ position: 'relative' } : {}`

**Review by i-abdul-raheem — CHANGES_REQUESTED** [review]

**Review by Haris-Siddiqui — COMMENTED** [review]
done

File: src/components/CreatePostWrapper/CreatePost.jsx (hunk 2 above)
Comment (Haris-Siddiqui) [reply src/components/CreatePostWrapper/CreatePost.jsx:8]: resolve conversation

File: src/components/CreatePostWrapper/Footer.jsx (hunk 4 above)
Comment (Haris-Siddiqui) [reply src/components/CreatePostWrapper/Footer.jsx]: done

**Review by i-abdul-raheem — APPROVED** [review]
Remember to change style to null if not implemented

**Pull request merged**